src/less/page-notice/base/page-notice.less

```diff
@@ -136,18 +136,19 @@ span[role="region"].page-notice {
 
 @media (prefers-color-scheme: dark) {
     .page-notice {
-        --page-notice-color: @color-text-default-dark-mode;
-        --page-notice-general-background-color: @color-grey1-dark-mode;
-        --page-notice-general-border-color: @color-grey1-dark-mode;
-        --page-notice-attention-background-color: #191f1b;
-        --page-notice-attention-border-color: #191f1b;
-        --page-notice-confirmation-background-color: #1b2027;
-        --page-notice-confirmation-border-color: #1b2027;
-        --page-notice-information-background-color: #2a191c;
-        --page-notice-information-border-color: #2a191c;
+        --page-notice-color: @color-black;
+        --page-notice-general-background-color: @color-grey6-dark-mode;
+        --page-notice-general-border-color: @color-grey6-dark-mode;
+        --page-notice-attention-background-color: @color-status-attention-dark-mode;
+        --page-notice-attention-border-color: @color-status-attention-dark-mode;
+        --page-notice-confirmation-background-color: @color-g6-dark-mode;
+        --page-notice-confirmation-border-color: @color-g6-dark-mode;
+        --page-notice-information-background-color: @color-status-information-dark-mode;
+        --page-notice-information-border-color: @color-status-information-dark-mode;
     }
 
     .page-notice svg {
+        .color(color-black);
```

Comment:
We shouldn't be creating a custom property here inside of the dark mode media query. Custom properties are created in the default/light mode rules, and then overridden here.

We can also remove `filter: none` from dark mode now as you removed the filter.

Reply:
Also, as Andrew mentioned, don't forget to run `npm run build` (there are some dist files that need generating due to that color primitive that was changed)

Reply:
re-rerunning `npm run build` didn't change the docs files... I'm not sure why they changed